src/lib/CLValue2/AccountHash.ts

```diff
@@ -1,6 +1,17 @@
-import { CLType, CLValue, ToBytes } from './Abstract';
```

Comment:
AccountHash is not CLType https://docs.rs/casper-types/1.0.1/casper_types/enum.CLType.html.
But I would keep it for now. Let's try to talk to core devs about adding it, as it's good idea to me.